vars/uiVeri5ExecuteTests.groovy

```diff
@@ -11,33 +11,21 @@ import static com.sap.piper.Prerequisites.checkScript
 @Field String STEP_NAME = getClass().getName()
 
 @Field Set GENERAL_CONFIG_KEYS = [
-    /**
-     * In case a `testRepository` is provided and it is protected, access credentials (as Jenkins credentials) can be provided with `gitSshKeyCredentialsId`. **Note: In case of using a protected repository, `testRepository` should include the ssh link to the repository.**
-     * @possibleValues Jenkins credentialId
-     */
+    /** @see seleniumExecuteTests */
     'gitSshKeyCredentialsId'
 ]
 @Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS.plus([
-    /**
-     * A map of environment variables to set in the container, e.g. [http_proxy:'proxy:8080'].
-     */
+    /** @see dockerExecute */
     'dockerEnvVars',
-    /**
-     * The name of the docker image that should be used. If empty, Docker is not used and the command is executed directly on the Jenkins system.
-     */
+    /** @see dockerExecute */
     'dockerImage',
-    /**
-     * Only relevant for Kubernetes case: Specifies a dedicated user home directory for the container which will be passed as value for environment variable `HOME`.
-     */
+    /** @see dockerExecute */
     'dockerWorkspace',
-    /**
-     * With `failOnError` the behavior in case tests fail can be defined.
-     * @possibleValues `true`, `false`
-     */
+    //TODO: transitive defaults
+    //TODO: transitive possible values
```

Comment:
@marcusholl: Can this be implemented easily?

Reply:
yes, we should implement this. Should not be that hard ... -> for now I would like to suggest to repeat the values here. Not nice, but we should not wait for that to be implemented.